easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -505,18 +498,42 @@ def to_str(x):
                 for key2, [def_val, _, _] in DEFAULT_CONFIG.items():
                     # only print parameters that are different from the default value
                     if key1 == key2 and val != def_val:
-                        ebtxt.append("%s = %s" % (key1, to_str(val)))
+                        ebtxt.append("%s = %s" % (key1, quote_str(val, esc_newline=True)))
                         printed_keys.append(key1)
             ebtxt.append("")
 
         # print other easyconfig parameters at the end
         for key, [val, _, _] in DEFAULT_CONFIG.items():
             if not key in printed_keys and val != self._config[key][0]:
-                ebtxt.append("%s = %s" % (key, to_str(self._config[key][0])))
+                ebtxt.append("%s = %s" % (key, quote_str(self._config[key][0], esc_newline=True)))
 
-        eb_file.write('\n'.join(ebtxt))
+        eb_file.write(self.wrap('\n'.join(ebtxt), width=120))
         eb_file.close()
 
+    
+    def wrap(self, text, width=80):
```

Comment:
we'll revisit `textwrap` vs this custom wrap function in an upcoming PR (to tackle the tasks mentioned in #1307)

for now, we'll look into getting this PR merged, since it fixes #1305